azure-governance-cloudwise/OMSAzureDashboards/OMSSQLDBAzureMonitoringSolution.omsview

```diff
@@ -0,0 +1,489 @@
+{
```

Comment:
not a template file. The OMS views need to be installed manually.Steps documented in the images/DeploymentGuide_and_Configuration_Guide_CloudWise.pdf > section 1.6